packages/voila/src/plugins.ts

```diff
@@ -22,14 +22,26 @@ import { KernelConnection } from '@jupyterlab/services/lib/kernel/default';
 
 import { ITranslator, TranslationManager } from '@jupyterlab/translation';
 
+import {
+  WidgetRenderer,
+  KernelWidgetManager
+} from '@jupyter-widgets/jupyterlab-manager';
+
 import {
   IJupyterWidgetRegistry,
   IWidgetRegistryData
 } from '@jupyter-widgets/base';
 
 import { VoilaApp } from './app';
 
-import { WidgetManager as VoilaWidgetManager } from './manager';
+import { Widget } from '@lumino/widgets';
+
+let resolveManager: (value: KernelWidgetManager) => void;
```

Comment:
I'm not entirely sure but does the `PromiseDelegate` help simplify this logic?

Reply:
Also using something like this at the module level can make it more difficult to reuse and decouple plugins later if needed (looks similar to https://github.com/voila-dashboards/voici/pull/38).

Reply:
Done!